walkthroughs/howto-k8s-multiple-listeners/app/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func startServer(port string) {
+	mux := http.NewServeMux()
+
+	mux.Handle("/ping", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {}))
+	mux.Handle("/", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintln(w, "Hello world")
```

Comment:
It would be nice if it spit out the port or something so people could tell this was all working.

Reply:
yeah i actually tested with that but didnt commit that version. thanks for actually reading this stuff and being thorough